.github/workflows/sass.yaml

```diff
@@ -1,94 +1,115 @@
 name: Sass
 
-on: [push, pull_request]
+on:
+  pull_request:
```

Comment:
@romaricpascal Yeah I did but I had to remove it 😆 

They're marked as mandatory checks via a **Branch protection rule** for the merge to be allowed. Can we discuss as a team, one for another PR rather than this one?

### Require status checks to pass before merging
<img width="613" alt="Status checks that are required" src="https://user-images.githubusercontent.com/415517/192243174-c7f1126b-1497-48d4-af74-fd16168fcea2.png">

Reply:
Oh, I see. It's probably worth a discussion and some background on why only the Sass tests are required (and not the Nunjucks or JavaScript ones 🤔 )

Reply:
All of the tests should be required… 'Run tests' is in that list of required status checks.

Reply:
Oh yeah, I'll mark this as resolved 😊